### ISSUE TITLE
Small improvement: Add legend to background map selection so that user knows how to toggle/untoggle maps

### ISSUE THREAD

**Issue by tifa365** [gislars/strassenraum-berlin#34]
First I was under the impression the background maps were not loading, since I simply clicked on each map to load, as you would on different categories of a website's menu. Then I figured you would have to untoggle/unselect a map by clicking again. Some people might be familiar with this behavior but for the common user it would be nice to have a short explanation, such as "Einzelkarte durch Klicken auswählen und deselektieren" or similar.

**Comment by tordans** [comment]
Yes, that feature is buggy. Will not fix it in this Version however, but during the upcoming rewriting of the app as a React/Gatsby Project.
